app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'users_id'
+  has_many :likes, foreign_key: 'users_id'
+  has_many :comments, foreign_key: 'users_id'
```

Comment:
[OPTIONAL] In your DB schema, there is no users_id column in the posts, likes, and comments tables. I recommend fixing the foreign_keys at the line (2,3 and 4) so the associations between models is correct.